Userland/Libraries/LibWeb/CSS/CSSFontFaceRule.cpp

```diff
@@ -31,8 +31,77 @@ CSSStyleDeclaration* CSSFontFaceRule::style()
 // https://www.w3.org/TR/cssom/#ref-for-cssfontfacerule
 String CSSFontFaceRule::serialized() const
 {
-    // FIXME: Implement this!
-    return "@font-face { /* FIXME: Implement CSSFontFaceRule::serialized()! */ }";
+    StringBuilder builder;
+    // The result of concatenating the following:
+
+    // 1. The string "@font-face {", followed by a single SPACE (U+0020).
+    builder.append("@font-face { "sv);
+
+    // 2. The string "font-family:", followed by a single SPACE (U+0020).
+    builder.append("font-family: "sv);
+
+    // 3. The result of performing serialize a string on the rule’s font family name.
+    builder.append(m_font_face.font_family());
+
+    // 4. The string ";", i.e., SEMICOLON (U+003B).
+    builder.append(";"sv);
+
+    // 5. If the rule’s associated source list is not empty, follow these substeps:
+    if (!m_font_face.sources().is_empty()) {
+        // 1. A single SPACE (U+0020), followed by the string "src:", followed by a single SPACE (U+0020).
+        builder.append("src: "sv);
+
+        // 2. The result of invoking serialize a comma-separated list on performing serialize a URL or serialize a LOCAL for each source on the source list.
+        for (size_t i = 0; i < m_font_face.sources().size(); i++) {
+            auto source = m_font_face.sources().at(i);
+            if (source.url.cannot_be_a_base_url()) {
+                builder.append("url("sv);
+                builder.append(source.url.to_string());
+                builder.append(")"sv);
+            } else {
+                builder.append("local("sv);
+                builder.append(source.url.to_string());
```

Comment:
This either needs to be:
```suggestion
                serialize_a_string(builder, source.url.to_string());
```
or, pull the "serialize a LOCAL" steps into a function in `CSS/Serialize.h`.